courses/free-courses-ur.md

```diff
@@ -57,6 +58,11 @@
 * [Java Programming in Urdu/Hindi](https://www.youtube.com/playlist?list=PLU4yvac0MJbKs78u32MyVgYFg9d-6DYGL) - OnlineUstaad
 
 
+### Next.js
+
+* [Master Next JS 14: Complete Next JS 14 Tutorial from Basics to Advanced in Hindi/Urdu with Projects & Interview Prep | The Techzeen](https://www.youtube.com/playlist?list=PL5OhSdfH4uDu6YJcHhmQLkwx4hPWyppos) - The Techzeen
```

Comment:
```suggestion
* [Master Next JS 14: Complete Next JS 14 Tutorial from Basics to Advanced in Hindi/Urdu with Projects & Interview Prep](https://www.youtube.com/playlist?list=PL5OhSdfH4uDu6YJcHhmQLkwx4hPWyppos) - The Techzeen
```
redundant

Reply:
Got it would consider this in future commits as well 👍